docs/design/values.md

```diff
@@ -963,11 +908,29 @@ pursued based on a real and concrete need, and the minimal extension is adopted.
 
 ## Alternatives considered
 
-TODO
+-   [No `var` introducer keyword](/proposals/p0339.md#no-var-introducer-keyword)
+-   [Name of the `var` statement introducer](/proposals/p0339.md#name-of-the-var-statement-introducer)
+-   [Colon between type and identifier](/proposals/p0339.md#colon-between-type-and-identifier)
+-   [Type elision](/proposals/p0339.md#type-elision)
+-   [Type ordering](/proposals/p0618.md#type-ordering)
+-   [Elide the type instead of using `auto`](/proposals/p0851.md#elide-the-type-instead-of-using-auto)
+-   [Value expression escape hatches](/proposals/p2006.md#value-expression-escape-hatches)
+-   [References in addition to pointers](/proposals/p2006.md#references-in-addition-to-pointers)
+-   [Syntax-free or automatic dereferencing](/proposals/p2006.md#syntax-free-or-automatic-dereferencing)
+-   [Exclusively using references](/proposals/p2006.md#exclusively-using-references)
+-   [Alternative pointer syntaxes](/proposals/p2006.md#alternative-pointer-syntaxes)
+-   [Alternative syntaxes for locals](/proposals/p2006.md#alternative-syntaxes-for-locals)
 
 ## References
 
--   Proposal
-    [#257: Initialization of memory and values](https://github.com/carbon-language/carbon-lang/pull/257)
--   Proposal
-    [#2006: Values, variables, and pointers](https://github.com/carbon-language/carbon-lang/pull/2006)
+-   [Proposal #257: Initialization of memory and values][p0257]
+-   [Proposal #339: `var` statement][p0339]
+-   [Proposal #618: `var` ordering][p0618]
+-   [Proposal #851: auto keyword for vars][p0851]
+-   [Proposal #2006: Values, variables, and pointers][p2006]
+
+[p0257]: /proposals/p0257.md
+[p0339]: /proposals/p0339.md
+[p0618]: /proposals/p0618.md
+[p0851]: /proposals/p0851.md
+[p2006]: /proposals/p2006.md
```

Comment:
I thought our convention was to link to the pull request for proposals, unless specifically trying to point to a specific file in the proposal?